src/styles/index.scss

```diff
@@ -1,3 +1,2 @@
-body {
-  margin: 0;
-}
+@import "variables";
```

Comment:
Ensure that the `variables` file exists in the same directory or provide the correct path if it's located elsewhere.